examples/tutorials/analysis-1d/cta_sensitivity.py

```diff
@@ -175,8 +175,8 @@
 )
 
 ax.loglog()
-ax.set_xlabel(f"Energy ({t['energy'].unit})")
-ax.set_ylabel(f"Sensitivity ({t['e2dnde'].unit})")
+ax.set_xlabel(f"Energy [{t['energy'].unit}]")
+ax.set_ylabel(f"Sensitivity [{t['e2dnde'].unit}]")
```

Comment:
There is the question, whether we should make the `DEFAULT_LABEL_TEMPLATE` public and use it here...maybe not...I guess it distracts from the content too much.

Reply:
It's also dangerous, because of how python globals work.

I would prefer functions, as these are "safer" for users.

How about the a function:

```
def format_label(quantity, unit=None, template=None, convention="divide")
```

that would also allow for the cases where we don't have units (and where the current approach would render something wrong like "foo / " or "foo []".